Source/LightboxTransition.swift

```diff
@@ -97,8 +97,16 @@ extension LightboxTransition : UIViewControllerAnimatedTransitioning {
               UIApplication.sharedApplication().keyWindow?.addSubview(screens.from.view)
           })
         } else {
-          transitionContext.completeTransition(true)
-          UIApplication.sharedApplication().keyWindow?.addSubview(screens.to.view)
+          if self.lightboxController.view.alpha < 0.97 && !self.presentingViewController {
```

Comment:
Totally random, but means that it's less than 1, but a bit bigger than 0.95 which is the final value.

I mean: The final number is 0.95, put 0.97 just in case there's a small error.